### ISSUE TITLE
Missing implementation for MParser_RE

### ISSUE THREAD

**Issue by basus** [ocaml/opam-repository#8000]
I just used `opam` to upgrade `MParser`. Now one of the projects I'm building using it gives me the following error:

```
ocaml setup.ml -build -j 4
Finished, 0 targets (0 cached) in 00:00:00.
+ ~/.opam/4.03.0/bin/ocamlfind ocamlopt -linkpkg -g -thread -I lib -linkpkg -thread -package yojson -package ulex -package threads -package tcpip -package str -package ppx_jane -package ppx_deriving.show -package ppx_deriving.eq -package ppx_deriving.enum -package ocamlgraph -package mparser.re -package mparser -package cstruct -package core -package camlp4.lib -package camlp4.extend -package camlp4 -package base64 -package async_parallel -package async -package ppx_sexp_conv -package cstruct.async -package cohttp.async -package cohttp -package async_extended lib/Frenetic_LP.cmx lib/frenetic.cmxa async/async.cmxa async/Frenetic_OpenFlow0x04_Plugin.cmx async/Frenetic_Shell.cmx frenetic/Dump.cmx frenetic/frenetic.cmx -o frenetic/frenetic.native
findlib: [WARNING] Interface topdirs.cmi occurs in several directories: ~/.opam/4.03.0/lib/ocaml/compiler-libs, ~/.opam/4.03.0/lib/ocaml
File "_none_", line 1:
Error: No implementations provided for the following modules:
         MParser referenced from ~/.opam/4.03.0/lib/mparser/lib_MParser_RE.cmxa(MParser_RE)
Command exited with code 2.
```

The MParser tarball linked to from the opam repo seems to contain the appropriate files.

**Comment by cakeplus** [comment]
A hotfix: https://github.com/ocaml/opam-repository/pull/8037.

**Comment by avsm** [comment]
fixed in #8037
